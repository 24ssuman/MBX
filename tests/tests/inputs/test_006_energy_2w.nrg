SYSTEM H2O
MOLECULE
MONOMER h2o
O      -1.822676041e+00   1.196579629e+00  -3.065692855e-01
H      -1.540849219e+00   2.811876585e-01  -2.693868828e-01
H      -1.287046524e+00   1.644910992e+00   3.498234010e-01
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
O      -4.644617442e+00   1.393015708e+00   4.174691364e-01
H      -3.736483474e+00   1.370498544e+00   9.483275726e-02
H      -5.144835789e+00   1.805604287e+00  -2.861691271e-01
ENDMON h2o
ENDMOL
ENDSYS
