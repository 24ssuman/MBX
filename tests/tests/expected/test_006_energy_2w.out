Energies without gradients:
system[....0]=     4.1433308471e+00    kcal/mol
Energies with gradients:
system[....0]=     4.1433308471e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O        -1.3929247746e+01    7.5943042462e-01    4.0367987676e+00
H        -1.4284099156e+00   -7.4429567438e-01   -8.6462641967e-01
H        -1.0076155236e+00    1.5166996569e+00    1.6199828714e-01
O        -3.3326236743e+00    1.4006611429e+00   -1.7187789928e+00
H         1.8849147431e+01   -1.7012555772e+00   -3.9791383391e+00
H         8.4874942820e-01   -1.2312399728e+00    2.3637466969e+00
